e2e/portalicious/pages/RegistrationActivityLogPage.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '@playwright/test';
+import { Page } from 'playwright';
+
+import BasePage from './BasePage';
+import TableComponent from './TableComponent';
+
+class RegistrationActivityLogPage extends BasePage {
+  readonly page: Page;
+  readonly table: TableComponent;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.table = new TableComponent(page);
+  }
+
+  async validateLastMessageSent(message: string) {
+    const table = new TableComponent(this.page);
+
+    const dropdownButton = await table.getCell(0, 0);
```

Comment:
I think some of this code could be replaced by a reusable `this.table.expandRow(0)`